lib/LaTeXML/Package/graphics.sty.ltxml

```diff
@@ -24,14 +24,17 @@ use LaTeXML::Util::Image;
 
 DefParameterType('GraphixDimension', sub {
     my ($gullet) = @_;
-    if ($gullet->ifNext(T_OTHER('!'))) {
-      $gullet->readToken();
+    $gullet->skipSpaces;
+    my $next = $gullet->readXToken;
+    # Wrong !?!?!
```

Comment:
curious if there are details :>

Reply:
just wondering if interpretation is correct